mmdet3d/datasets/det3d_dataset.py

```diff
@@ -113,18 +113,15 @@ def __init__(self,
                 ori_label = self.METAINFO['classes'].index(name)
                 self.label_mapping[ori_label] = label_idx
 
-            self.num_ins_per_cat = {name: 0 for name in metainfo['classes']}
+            self.num_ins_per_cat = [0] * len(metainfo['classes'])
```

Comment:
Why do we need to revise this part?

Reply:
https://github.com/open-mmlab/mmdetection3d/issues/2764 because this bug, revise this can accelerate all dataset setup.